pyacq/core/stream.py

```diff
@@ -80,9 +82,14 @@
 class OutputStream:
     """A OutputStream is a helper class to send data.
     """
-    def __init__(self, spec = {}):
+    def __init__(self, spec = {}, node = None, name = None):
         self.configured = False
         self.spec = spec # this is a priori stream params, and must be change when Node.configure
+        if node:
```

Comment:
Is a Node true or false? Is None true or false? I can never remember.
Better to be explicit:  `if node is None`